build/lib/rollup.js

```diff
@@ -39,8 +39,8 @@ async function rollupModule(options) {
         });
         const result = generatedBundle.output[0];
         result.code = result.code + '\n//# sourceMappingURL=' + path.basename(outputMapName);
-        await fs.promises.writeFile(outputFilePath, result.code);
-        await fs.promises.writeFile(outputMapPath, result.map);
+        await fs.promises.writeFile(outputFilePath, result.code.toString());
+        await fs.promises.writeFile(outputMapPath, result.map ? result.map.toString() : '');
```

Comment:
Should we even be writing these map files if there isn't any content? Seems better to just skip that entirely.